scripts/docker/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:bionic
```

Comment:
I am thinking on specifying the following options:
- [VOLUME /CodeCompass](https://docs.docker.com/engine/reference/builder/#volume)
- [ENTRYPOINT ["start-devenv", "build"]](https://docs.docker.com/engine/reference/builder/#entrypoint)
- [EXPOSE 8080](https://docs.docker.com/engine/reference/builder/#expose)

Reply:
I didn't add `VOLUME` because it is used for files from the container to the host. We don't have such files. I also haven't used `EXPOSE` because that will be necessary for the web service. This is just a CodeCompass devenv Docker image.